src/Selector/Xpath/Manipulator.php

```diff
@@ -43,6 +48,21 @@ public function prepend($xpath, $prefix)
             $prefix = '('.$prefix.')';
         }
 
+        // Get all xpath strings.
```

Comment:
We should add a documentation why we are doing the replacement here: Support "|" in actual strings